docs/tools/aelf-studio-extention/index.md

```diff
@@ -0,0 +1,163 @@
+| Deploy Smart Contracts    | Deploy smart contracts to the aelf testnet.                                                   | `Deploy aelf smart contract`               |
+| Check Transaction Status  | Track deployment and blockchain transaction statuses directly from the extension.             | `Check status of transaction`              |
+| AI Audits                 | Use AI tools to audit smart contracts for vulnerabilities and potential issues.               | `Audit aelf smart contract`                |
+| Deploy from Local         | Deploy locally built smart contracts to the aelf blockchain.                                  | `Deploy aelf smart contract from a local dll` |
+
+## 3. Setting Up aelf-Studio
+
+- **VS Code**: Make sure you have VS Code installed. You can download it from [here](https://code.visualstudio.com/).
+- **.NET SDK**: You also need the .NET SDK to write smart contracts. Download it from [here](https://dotnet.microsoft.com/en-us/download).
+
+### Steps to install aelf-Studio:
+
+- **Open VS Code:**
+   - Start the visual studio code program.
+
+   ![vs-code-aelf-studio](/img/vs-code-aelf-studio.png)
+
+- **Install the aelf-Studio Extension:**
+   - On the left side of VS Code, click the `Extensions` icon.
+
+   ![click-extention-aelf-studio](/img/click-extention-aelf-studio.png)
+
+
+   - In the search box, type **“aelf-studio”**.
+
+   ![search-extention-aelf-studio](/img/search-extention-aelf-studio.png)
+
+
+   - Find the extension and click **Install**.
+
+   ![click-install-aelf-studio](/img/click-install-aelf-studio.png)
+
+
+   - If VS code asks to reload, click reload.
+
+- **Check if it’s installed**:
+
+   - `aelf` icon will be visible at the bottom right corner of the screen after successful installation.
+
+   ![bottom-icon-aelf-studio](/img/bottom-icon-aelf-studio.png)
+
+## 4. How to Use aelf Studio
+
+### Create a new project:
+
+
+- **Make a new folder:**
+  - Open the terminal in VS Code (Click **Terminal** at the top and choose **New Terminal**). 
+  - Type `mkdir hello-world` and press enter. This makes a new folder called "hello-world".
+  
+    ![mk-dir-aelf-studio](/img/mk-dir-aelf-studio.png)
+
+  - Type `cd hello-world` and press enter. This will change the directory to "hello-world" folder.
+
+    ![cd-aelf-studio](/img/cd-aelf-studio.png)
+
+  - Then type `code .` to open that folder in VS Code.
+
+    ![code-aelf-studio](/img/code-aelf-studio.png)
+
+- **Create a new smart contract:**
+  - In the terminal, type `dotnet new aelf -n HelloWorld` and press enter. This will create a `HelloWorld` smart contract template project.
+
+    ![create-hello-world-aelf-studio](/img/create-hello-world-aelf-studio.png)
+
+- **Build the contract:**
+  - Go to the "src" folder by typing `cd src`.
+  - Then type `dotnet build` to build (compile) the smart contract.
```

Comment:
Please use aelf-Studio’s build feature instead of dotnet build and update screenshot

Reply:
Updated changes